src/components/common/Input/Input.styled.tsx

```diff
@@ -0,0 +1,28 @@
+import styled from '@emotion/styled';
+
+import { Props } from '.';
+import { colors } from '@/styles/global';
+
+export const StyledInputContainer = styled.input<Pick<Props, 'isValid'>>`
+  box-sizing: border-box;
+  width: 100%;
+  padding: 8px;
+  height: 40px;
+  color: black;
+  background-color: white;
+  font-size: 14px;
+  border: solid 1px;
+  border-radius: 4px;
+  outline: none;
+  ${({ isValid }) => 'border-color : ' + (isValid ? `${colors.GY1}` : `${colors.red}`)};
```

Comment:
아래 방식으로도 가능한데요. 어때요?

```suggestion
  border-color: ${({ isValid }) => (isValid ? colors.GY1 : colors.red)};
```

Reply:
수정해주신 방법이 훨씬 깔끔해 보입니다!!

Reply:
3903b6ce8bd50dd3f412f4b98a237a863351854b
수정했습니다~